keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -138,7 +140,6 @@ def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
         image = None
         if "image" in kwargs:
             image = kwargs.get("image")
-        bbox_dtype = bounding_boxes.dtype
         if self.bounding_box_format is None:
             raise ValueError(
                 "bounding_box_format has not been specified during layer "
```

Comment:
lets provide more actionable feedback to users.  Errors should be self correction; ideally without a trip to the docs.

```
"`RandomRotation()` was called with bounding boxes, but no `bounding_box_format` was specified in the constructor.  Please specify a bounding box format in the constructor.  i.e. `RandomRotation(bounding_box_format='xyxy')`
```

Reply:
Done